src/checkers/ast.rs

```diff
@@ -1867,6 +1867,7 @@ where
                     || self.settings.enabled.contains(&RuleCode::F525)
                     // pyupgrade
                     || self.settings.enabled.contains(&RuleCode::UP030)
+                    || self.settings.enabled.contains(&RuleCode::UP032)
```

Comment:
(This was causing the fixtures to be a no-op per your question in the PR summary @colin99d.)

Reply:
Thank you!!